api/api.yaml

```diff
@@ -132,6 +132,26 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/routes:
```

Comment:
Should this be '/control/routes' or '/control/device' or '/control/routers'?

Reply:
change it to /control/routers